src/index.html

```diff
@@ -12,7 +12,41 @@
       href="styles/index.scss"
     />
```

Comment:
The `href` attribute is linking to an SCSS file (`styles/index.scss`). Browsers cannot interpret SCSS files directly. You need to compile your SCSS into a CSS file and link to that CSS file instead.